docs/docs/1.6.x/documentation/requirements.md

```diff
@@ -0,0 +1,30 @@
+# Requirements
+
+This page expose the different requirements to run Kuma.
+
+## Kubernetes
+
+Kuma is validated against Kubernetes `1.19.x`, `1.20.x`, `1.21.x` and `1.22.x`.
+
+## Envoy
+
+Versions of envoy supported are: `~1.21.1` which means `>=1.21.1` and `<1.22.0`.
+
+## Sizing your control-plane
+
+TL;DR A control-plane with 4vCPU and 2GB of memory will be able to accommodate more than 1000 data planes.
```

Comment:
`data planes` -> `data plane proxies`.
`TL;DR` -> maybe something a little more formal? `In short`? `Overall`? @michaelbeaumont any good idea?